src/redux/categories/categories.jsx

```diff
@@ -0,0 +1,37 @@
+// Actions
+
+const LOAD = 'my-app/categories/LOAD';
+const STATUS = 'my-app/categories/STATUS';
+
+// Define an initial state value for the app
+
+const initialState = [
+  {
+    status: 'Under construction',
+  },
+];
+
+// Reducer
+
+export const categoriesReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case LOAD:
+      return [
+        ...state, action.payload,
+      ];
+    case STATUS:
+      return [
+        ...state, action.payload,
+      ];
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] Please know that you should be using `@reduxjs/toolkit` here as well, as I explained before what you used is the old way of implementing a reducer you need to use the new way using the toolkit, kindly check the lesson again and see how to use the new proper way of `@reduxjs/toolkit`.